core/stdlib/std.ncl

```diff
@@ -1971,7 +1971,7 @@
         "%
       = fun field content r =>
         let r =
-          if %has_field% field r then
+          if %has_field_all% field r then
```

Comment:
Did this sneak in from the other pr?

Reply:
Yes, and it's obsolete. I'll revert this change, thanks for the catch.

Reply:
Ah, it's outdated in fact, I did that already. Resolving this conversation